### PR TITLE
Added title for show to first 

### PR DESCRIPTION
**Pull request by abgaryanharutyun**
Added new option for add title above to icon default false

### CODE REVIEW

File: src/utils/dom.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ const sweetHTML = `
    <div class="${swalClasses.icon} ${iconTypes.error}">
      <span class="swal2-x-mark"><span class="swal2-x-mark-line-left"></span><span class="swal2-x-mark-line-right"></span></span>
    </div>
+   <h2 class="${swalClasses.titleFirst}" id="${swalClasses.titleFirst}"></h2>
```
Comment (toverux) [inline src/utils/dom.js]: I don't really like that solution :/ It would be better to invert elements programmatically or use CSS.

**Comment by abgaryanharutyun** [comment]
@toverux thanks for comment i will take a look

**Comment by limonte** [comment]
This will be done a little later this week in scope of #800 

@abgaryanharutyun subscribe to #800 to be informed. I'll post the jsfiddle with your case there after required changes are done.

**Comment by limonte** [comment]
Hello again @abgaryanharutyun 

Here you go with examples:

- JS-way: https://jsfiddle.net/ad3quksn/333/
- CSS-way: https://jsfiddle.net/ad3quksn/334/

**Comment by abgaryanharutyun** [comment]
@limonte awesome thanks

**Pull request closed without merging**